### DISCUSSION TITLE
How to rename a group in the new reworked node UX version?

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#381] [Q&A]
![image](https://user-images.githubusercontent.com/36077492/185413103-50ed2c85-8d18-4196-a709-6ddc86319db5.png)

The "NPR Diffuse" has been renamed as "Diffuse". But in the right property panel, it is still named as "NPR Diffuse"

**Comment by HSDSZ** [comment]
Found it. Reloading the malt pipeline do the job.

Can we still have the old way : simply reconnecting any one of the nodes.

**Reply by pragma37** [reply]
Should be fixed now. 👍
